state-chain/pallets/cf-threshold-signature/src/lib.rs

```diff
@@ -157,6 +160,7 @@ pub enum KeyRotationStatus<T: Config<I>, I: 'static = ()> {
 		ceremony_id: CeremonyId,
 		response_status: KeyHandoverResponseStatus<T, I>,
 		receiving_participants: BTreeSet<T::ValidatorId>,
+		#[skip_name_expansion]
```

Comment:
is there a reason we do this on some like this but not on others like `CeremonyId` above?